src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -2234,16 +2234,40 @@ public void setMaxHintWindow(int ms)
         DatabaseDescriptor.setMaxHintWindow(ms);
     }
 
+    public int getMaxHintsSizePerHostInMb()
+    {
+        return DatabaseDescriptor.getMaxHintsSizePerHostInMb();
+    }
+
+    public void setMaxHintsSizePerHostInMb(int value)
+    {
+        DatabaseDescriptor.setMaxHintsSizePerHostInMb(value);
+    }
+
     public static boolean shouldHint(Replica replica)
     {
         return shouldHint(replica, true);
     }
 
+    /**
+     * Determines whether a hint should be stored or not.
```

Comment:
I guess a better name for this method would have been `shouldStoreHint()`

Reply:
I agree with you that `shouldStoreHint` is a better name. The name has been there and it is a public function. So I am being conservative to not change it. At the end, `shouldHint` can describe the function too.